docs/podman-system-df.1.md

```diff
@@ -0,0 +1,17 @@
+% podman-system-df(1) podman
+
+## NAME
+podman\-system\-df - Show podman disk usage
+
+## SYNOPSIS
+**podman system df**
+[**-help**|**--h**]
+
+## DESCRIPTION
+Show podman disk usage
+
+## SEE ALSO
+podman-system(1)
```

Comment:
Even thought it's an easy command, it would be nice to have an example here so they'll know what to expect to see.